error/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub enum Error_type {
     PSM_OK_NO_PROGRESS = 1,
     PSM_PARAM_ERR = 3,
     PSM_NO_MEMORY = 4,
-    PSM_INIT_NOT_INIT = 5, 
+    PSM_INIT_NOT_INIT = 5,
```

Comment:
Ah yeah. I'd just be on the lookout to see if we need the explicit numbering. In rust, you almost certainly won't but there might be some kind of voodoo significance of these digits. Again, thats probably a code smell but whatevs.

I guess yeah roll with it for now.